src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,28 @@
+package duke.command;
+
+import duke.task.TaskList;
+import duke.util.Storage;
+
+/**
+ * Represents the a generic Command that will be extended from.
+ * The method <code>execute</code> should be implemented and
+ * called to run any command's behaviours.
+ *
+ * @author  Hardy Shein
+ * @version 0.1
+ */
+public abstract class Command {
+
+    /**
+     * Abstract execute method for all Command subclasses.
+     * @param taskList to access collection of tasks.
+     * @param storage to access save-load functionality.
+     */
+    public abstract void execute(TaskList taskList, Storage storage);
+
+    /**
+     * Abstract method for all Command subclasses.
+     * @return boolean indicating if command is an exit command.
+     */
+    public abstract boolean isExit();
```

Comment:
Just a personal preference, perhaps leave a blank line here too for consistency, since there's a blank line too at the top. Otherwise, very neat code and coding style!